src/tests/ftest/util/run_utils.py

```diff
@@ -465,6 +465,8 @@ def run_remote(log, hosts, command, verbose=True, timeout=120, task_debug=False,
     Returns:
         CommandResult: groups of command results from the same hosts with the same return status
     """
+    if timeout < 240:
+        timeout = 240
```

Comment:
This is problematic for callers that intentionally set a low timeout. E.g. if a caller does `timeout=20` it will now be 240. Which could lead many tests to timeout overall before the command times out, which means we don't have enough time to cleanup. Or cleanup commands themselves that set a low timeout will take longer, etc.

Also, there are callers that do `timeout=None`, in which case `timeout < 240` raises an exception

Reply:
Removed